gengokit/template/NAME-service/cmd/NAME/main.gotemplate

```diff
@@ -37,12 +37,30 @@ func main() {
 	os.Exit(submain())
 }
 
+type headerSeries []string
```

Comment:
This seems less intuitive to me than a `map[string][]string`, but that's picking nits, I think, because this is a straightforward enough implementation.